site/static/compare/script.js

```diff
@@ -433,6 +387,70 @@ app.component('test-cases-table', {
 </div>
 `
 });
+
+const SummaryPercentValue = {
+    props: {
+        value: Number,
+    },
+    template: `
+<span>{{ value.toFixed(2) }}%</span>
+`
+};
+const SummaryRange = {
+    props: {
+        range: Array,
+    },
+    template: `
+<div v-if="range.length > 0">
+  [<SummaryPercentValue :value="range[0]" />, <SummaryPercentValue :value="range[1]" />]    
+</div>
+<div v-else>-</div>
+`, components: {SummaryPercentValue}
+};
+const SummaryCount = {
+    props: {
+        cases: Number,
+        benchmarks: Number
+    },
+    template: `
+<span :title="cases + ' test case(s), ' + benchmarks + ' unique benchmark(s)'">{{ cases }} ({{ benchmarks }})</span>
+`
+};
+
+app.component('summary-table', {
+    props: ['cases'],
+    template: `
+<table class="summary-table">
+    <thead>
+      <th>Range</th>
+      <th>Mean</th>
+      <th>Count</th>
+      <th><!-- icon --></th>
```

Comment:
I personally don't find showing the icon last to be very intuitive. I would expect things that label the current row to be first, but I do understand the temptation to show the interesting data sooner to the user as they (presumably) read left to right.

Reply:
![image](https://user-images.githubusercontent.com/4539057/184938974-7b485a55-fe8a-4325-912f-41db79c86277.png)

I don't have a strong opinion here, but as you said, I wanted to keep the most important data first, at the left.